woodstock/voting/templates/vote.html

```diff
@@ -49,11 +49,19 @@
             {% else %}
               <img class="img-circle avatar" src="{% static 'img/shy.png' %}" />
             {% endif %}
-            <a href="https://mozillians.org/en-US/search/?q={{ mozillian.email }}"
-               target="_blank"
-               class="btn btn-small visible-desktop">
-              Mozillians Profile
-            </a>
+            {% if mozillian.mozillian_username %}
```

Comment:
You could avoid writing twice the same code by limiting the ``if/else`` case to the ``<a>`` element.